vyhledavani-reseni-1.php

```diff
@@ -0,0 +1,121 @@
+<?php
```

Comment:
Proč to řešení nedat do dalšího commitu, aby byl i vidět diff?

Reply:
protoze chci pak jednoduse prepinat mezi soubory.. a ten diff je jen v tom reseni nahore, coz jsou 2-3 radky.. prislo mi to zbytecne